src/compiler/transformers/static-to-meta/component.ts

```diff
@@ -18,14 +18,28 @@ import { parseStringLiteral } from './string-literal';
 import { setComponentBuildConditionals } from '../component-build-conditionals';
 import ts from 'typescript';
 
+/**
+ * Given an instance of typescript's IR for a class decoration ({@see ts.ClassDeclaration})
```

Comment:
Nit - can we update the first usage of this acronym to include its definition for future readers?

```suggestion
Given an instance of Typescript's Intermediate Representation (IR) for a class decoration ({@see ts.ClassDeclaration})
```